internal/testutil/testutil.go

```diff
@@ -89,11 +90,22 @@ func testContext() (Context, error) {
 		return tc, noProjectID
 	}
 
-	pkgs, err := packages.Load(&packages.Config{Mode: packages.NeedName}, "github.com/GoogleCloudPlatform/golang-samples")
+	cfg := &packages.Config{
+		Mode:  packages.NeedName | packages.NeedFiles,
+		Tests: true,
+	}
+	pkgs, err := packages.Load(cfg, "github.com/GoogleCloudPlatform/golang-samples")
 	if err != nil {
-		return tc, fmt.Errorf("Could not find golang-samples: %v", err)
+		return tc, fmt.Errorf("could not find golang-samples: %v", err)
+	}
+	for _, pkg := range pkgs {
+		if len(pkg.GoFiles) > 0 && strings.HasSuffix(pkg.GoFiles[0], ".go") {
```

Comment:
Which other kinds of files does this categorize as GoFiles?

Reply:
There is one package with a file without the `.go` extension.